controllers.go

```diff
@@ -133,6 +133,13 @@ func imageHandler(w http.ResponseWriter, r *http.Request, buf []byte, operation
 	// Expose Content-Length response header
 	w.Header().Set("Content-Length", strconv.Itoa(len(image.Body)))
 	w.Header().Set("Content-Type", image.Mime)
+	if image.Mime != "application/json" {
+		meta, err := bimg.Metadata(image.Body)
+		if err == nil {
+			w.Header().Set("X-Image-Width", strconv.Itoa(meta.Size.Width))
+			w.Header().Set("X-Image-Height", strconv.Itoa(meta.Size.Height))
+		}
+	}
```

Comment:
That's fine. Instead of this, we could simply support an optional query param `return_size=true` or HTTP request header `Imaginary-Return-Size: true` in order to enable this feature. Also, this can be by default enabled for all requests if explicitly enabled via command flags, e.g: `imaginary -p 8080 -return-size`.